src/AGCertificateVerifier.h

```diff
@@ -189,11 +264,11 @@ class AGCertificateVerifier {
 
     AGVerifyResult verifyPins(const std::string &dnsName, STACK_OF(X509) *certChain);
 
-    AGVerifyResult verifyRevocations(STACK_OF(X509) *certChain);
+    AGVerifyResult verifyCrlSetsStatus(STACK_OF(X509) *certChain);
```

Comment:
What are the pre-requisites for this method to work properly?
What if I didn't call updateCrlSets prior to this call?